gno.land/cmd/gnoland/genesis_txs_add.go

```diff
@@ -5,26 +5,34 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"github.com/gnolang/gno/gno.land/pkg/gnoland"
 	"github.com/gnolang/gno/tm2/pkg/bft/types"
 	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/crypto"
 	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
 var (
+	errInvalidTxsPath     = errors.New("invalid transactions path")
 	errInvalidTxsFile     = errors.New("unable to open transactions file")
 	errNoTxsFileSpecified = errors.New("no txs file specified")
 )
 
+var (
+	genesisDeployAddress = crypto.MustAddressFromString("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5") // test1
+	genesisDeployFee     = std.NewFee(50000, std.MustParseCoin("1000000ugnot"))
+)
+
 // newTxsAddCmd creates the genesis txs add subcommand
 func newTxsAddCmd(txsCfg *txsCfg, io commands.IO) *commands.Command {
 	return commands.NewCommand(
 		commands.Metadata{
 			Name:       "add",
-			ShortUsage: "txs add <tx-file ...>",
+			ShortUsage: "txs add <tx-path ...>",
 			ShortHelp:  "imports transactions into the genesis.json",
-			LongHelp:   "Imports the transactions from a tx-archive backup to the genesis.json",
+			LongHelp:   "Imports the transactions from a given transactions sheet, or package directory, to the genesis.json",
```

Comment:
Shall we be more specific? maybe is worth it to have two different commands one for each case, something like `txs add sheet` and `txs add packages`

In the future, you could even load several sheets from a folder, for example.

Reply:
I'm all for having 2 different commands, it's easier to manage 🤷‍♂️ 

I'm also not a huge fan of a command being "magical", as in it can parse frogs and unicorns under the hood